File: DSA/LakshaKundra_590013237/Day_3_Question_2.py
def contains_duplicate(nums):
    seen = set()  
    for num in nums:
        if num in seen:
            return True 
        seen.add(num)
    return False  
print(contains_duplicate([1, 2, 3, 4])) 
print(contains_duplicate([1, 2, 3, 2])) 
print(contains_duplicate([5, 5, 5]))    

File: DSA/Ayush_Mishra_590014844/Day_3_Question_2.c
#include <stdio.h>
#include <stdbool.h>
bool containsDuplicate(int* nums, int n) {
    int freq[10001] = {0};
    for (int i = 0; i < n; i++) {
        if (freq[nums[i]] > 0)
            return true;
        freq[nums[i]]++;
    }
    return false;
}

int main() {
    int nums[] = {1, 2, 3, 2};
    int n = sizeof(nums) / sizeof(nums[0]);
    if (containsDuplicate(nums, n))
        printf("Output: true\n");
    else
        printf("Output: false\n");
    return 0;
}
